src/components/SeekHelp/index.scss

```diff
@@ -12,6 +12,7 @@
     font-family: $mainFontFamily;
     font-weight: bold;
     margin-top: 3rem;
+    padding-top: 0.5rem;
```

Comment:
I suggest to put it in a variable, that would make it easy to change in one place. You can call it `headerPaddingTop` for example